sunpy/data/manager/cache.py

```diff
@@ -0,0 +1,68 @@
+from sunpy.util.util import hash_file
+
+
+class Cache:
+    def __init__(self, downloader, storage):
+        self._downloader = downloader
+        self._storage = storage
+
+    def download(self, urls, redownload=False):
+        """
+        Downloads the files from the urls
+
+        Parameters
+        ----------
+        urls: List
+        list of urls
+        redownload: `bool`
+        Whether to skip cache and redownload
+        """
+        # TODO: Expiry time
+        # XXX: Expiry time cache level or download level?
```

Comment:
What is the difference here? The time that the file was entered into the cache or the time the file was downloaded?

Reply:
No. That has to be decided too.

I was talking about having an expiry time per download or per cache. Like so

```python
c = Cache(expiry='10m')
c.download(stuuff)
```
vs

```python
c = Cache()
c.download(stuff, expiry = '10m')
c.download(stuff, expiry='5m')
```

I vote for the former because it is a more intuitive API.

Reply:
I would also vote the former. If there is a use case for expiring files at different times, we can think about it but I don't have one in my head.

Reply:
Ping @Cadair

Reply:
I think per-cache will be fine.